docs/collections/catalog/prefect-metricflow.yaml

```diff
@@ -0,0 +1,5 @@
+collectionName: prefect-metricflow
+author: Alessandro Lollo
+authorUrl: https://github.com/AlessandroLollo
+documentation: https://alessandrolollo.github.io/prefect-metricflow/
+iconUrl: https://github.com/transform-data/metricflow/raw/main/assets/MetricFlow_logo.png
```

Comment:
Can you add a square image with a transparent background for MetricFlow to this repo in the same location as the other collections images?

Reply:
Hey @desertaxle 👋 
I've updated `prefect-transform` in order to backport the new template!
Feel free to take a look [here](https://github.com/AlessandroLollo/prefect-transform)

Reply:
Ouch, I think I tagged the wrong Victoria in the last commit message 😅